Gems/Atom/RPI/Code/Include/Atom/RPI.Reflect/Image/Image.h

```diff
@@ -22,6 +22,20 @@ namespace AZ
     {
         class ImageAsset;
 
+        static const int s_TotalSupportedImageExtensions = 10;
+        static const char* s_SupportedImageExtensions[s_TotalSupportedImageExtensions] = {
+            "*.tif",
+            "*.tiff",
+            "*.png",
+            "*.bmp",
+            "*.jpg",
+            "*.jpeg",
+            "*.tga",
+            "*.gif",
+            "*.dds",
+            "*.exr"
+        };
+
```

Comment:
Do these need to move out of ImageProcessingAtom, or just to a public header in that gem? They seem a little specific to put in Image.h, and GradientSignal already has a dependency on ImageProcessingAtom.Headers.

Reply:
Also, suggestion:  Could do this instead...
```
static const char* s_SupportedImageExtensions[] = { 
...
};
static constexpr int s_TotalSupportedImageExtensions = AZ_ARRAY_SIZE(s_SupportedImageExtensions);
```

Reply:
Good call I like that better. I noticed that `ImageProcessingAtom.Headers`, but it is currently empty. I'll include that header in there. And will also update to use the `AZ_ARRAY_SIZE` instead, thanks!